### PR TITLE
🎨Autoscaling: chunk prepulled image AWS EC2 tags

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

during early testing, this was identified as a problem. AWS EC2 Tags take a maximum of 256 characters. When pre pulling images in buffer EC2, a tag with the pre-pulled images is set on the instance for fast retrieval.
Since the list of pre pulled images can be quite long now this is chunked.

### driving tests:
```services/autoscaling/tests/unit/test_utils_buffer_machines_pool_core.py```

## Related issue/s
- fixes https://github.com/ITISFoundation/osparc-simcore/issues/6230

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
Cool thanks!

File: services/autoscaling/src/simcore_service_autoscaling/utils/buffer_machines_pool_core.py [hunk 1]
```diff
@@ -36,4 +36,49 @@ def get_deactivated_buffer_ec2_tags(
...
+
+def load_pre_pulled_images_from_tags(tags: EC2Tags) -> list[DockerGenericTag]:
+    # AWS Tag values are limited to 256 characters so we chunk the images
+    if PRE_PULLED_IMAGES_EC2_TAG_KEY in tags:
+        # read directly
+        return json_loads(tags[PRE_PULLED_IMAGES_EC2_TAG_KEY])
+
+    assembled_json = "".join(
+        map(
+            itemgetter(1),
```
Comment (pcrespov) [inline services/autoscaling/src/simcore_service_autoscaling/utils/buffer_machines_pool_core.py:71]: your mind shouts out c++ ;-)

**Review by YuryHrytsuk — APPROVED** [review]

File: services/autoscaling/src/simcore_service_autoscaling/constants.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import re
+from typing import Final
+
+from aws_library.ec2._models import AWSTagKey, AWSTagValue, EC2Tags
+from pydantic import parse_obj_as
+
+BUFFER_MACHINE_PULLING_EC2_TAG_KEY: Final[AWSTagKey] = parse_obj_as(
+    AWSTagKey, "pulling"
```
Comment (pcrespov) [inline services/autoscaling/src/simcore_service_autoscaling/constants.py:8]: Nice that you inter-relate all concepts

**Review by pcrespov — APPROVED** [review]
thx for such a careful work!

**Review by giancarloromeo — APPROVED** [review]

File: services/autoscaling/src/simcore_service_autoscaling/utils/buffer_machines_pool_core.py (hunk 1 above)
Comment (sanderegg) [reply services/autoscaling/src/simcore_service_autoscaling/utils/buffer_machines_pool_core.py:71]: thanks

**Review by mrnicegyu11 — APPROVED** [review]
thx, lgtm

**Pull request merged**